assets/stylesheets/_components.scss

```diff
@@ -25,14 +25,12 @@
 @import 'components/forms/form-text-input/style';
 @import 'components/forms/form-text-input-with-affixes/style';
 @import 'components/forms/form-toggle/style';
-@import 'components/token-field/style';
```

Comment:
These styles are now loaded directly from the component's JSX file, same way as we load the main stylesheet (`import "./style.scss";`).